docs/develop/install_locally.md

```diff
@@ -35,35 +38,67 @@ have been symlinked inside it.
 
 ## Setup the database, Elasticsearch, Redis and RabbitMQ
 
-We need to initialize the database, the indices and so on. For this, we can use
-the `services` command. Note that this command is only needed once. Afterwards, you
-can stop (not destroy) these services and start again, and your data will still be there.
+We need to initialize the database, the indices and so on. For this, we use
+the `services` command. The first time this command is run, the services will be
+setup correctly and the containers running them will even restart upon a reboot
+of your machine. If you stop and restart those containers, your data will still
+be there. Upon running this command again, the initial setup is skipped.
 
+
+``` bash
+invenio-cli services
+```
 ``` console
-$ invenio-cli services
 Making sure containers are up...
-Creating database...
+Creating network "february-release-2_default" with the default driver
+Creating february-release-2_cache_1 ... done
```

Comment:
Minor comment not to be addressed now: Using a more generic name like `my-project` wouldn't be better than a specific `february-release`? Just for the ocd of it when we enter March :P

Reply:
Yes, maybe something like "My Digital Repository" eventually. Made a ticket for it to address it later: https://github.com/inveniosoftware/docs-invenio-rdm/issues/24